src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipsController.cs

```diff
@@ -25,18 +28,44 @@ public ApprenticeshipsController(IMediator mediator, ILogger<ApprenticeshipsCont
 
         [HttpGet]
         [Route("{providerId}")]
-        public async Task<IActionResult> GetApprenticeships(uint providerId)
+        public async Task<IActionResult> GetApprenticeships(uint providerId, [FromQuery]int pageNumber = 0, [FromQuery]int pageItemCount = 0, [FromQuery]string sortField = "", [FromQuery]bool reverseSort = false)
         {
             try
             {
-                var response = await _mediator.Send(new GetApprenticeshipsRequest {ProviderId = providerId});
+                var response = await _mediator.Send(new GetApprenticeshipsRequest
+                {
+                    ProviderId = providerId, 
+                    PageNumber = pageNumber, 
+                    PageItemCount = pageItemCount, 
+                    SortField = sortField,
+					ReverseSort = reverseSort
+                });
 
                 if (response == null)
                 {
                     return NotFound();
                 }
 
-                return Ok(response.Apprenticeships);
+                //TODO: Remove this mapping once we have consolidated the old Types with the new API types
+                var mappedApprenticeships = response.Apprenticeships.Select(x => new ApprenticeshipDetails
+                {
+                    ApprenticeFirstName = x.ApprenticeFirstName,
```

Comment:
We haven't prefixed the first or last names with "Apprentice" elsewhere, so it's a bit ugly here. Can we remove it? Also "Planned" from Start and EndDates. Seems an unnecessary prefix.

Reply:
Seems like these have a added beforehand and I've just carried it on. I'll make the changes now to both the api type and the general type.